src/psycopmlutils/timeseriesflattener/flattened_dataset.py

```diff
@@ -178,24 +190,66 @@ def add_predictors_from_list_of_argument_dictionaries(
 
         self.df = self.df_aggregating.drop(self.pred_time_uuid_col_name, axis=1)
 
-    def _create_flattened_df_wrapper(self, kwargs_dict: Dict) -> DataFrame:
-        """Wrap create_flattened_df with kwargs for multithreading pool.
+    def _flatten_temporal_values_to_df_wrapper(self, kwargs_dict: Dict) -> DataFrame:
+        """Wrap flatten_temporal_values_to_df with kwargs for multithreading pool.
 
         Args:
             kwargs_dict (Dict): Dictionary of kwargs
 
         Returns:
             DataFrame: DataFrame generates with create_flattened_df
         """
-        return self.create_flattened_df_for_value(
+        return self.flatten_temporal_values_to_df(
             prediction_times_with_uuid_df=self.pred_times_with_uuid,
             id_col_name=self.id_col_name,
             timestamp_col_name=self.timestamp_col_name,
             pred_time_uuid_col_name=self.pred_time_uuid_col_name,
             **kwargs_dict,
         )
 
-    def add_outcome(
+    def add_age(
+        self, date_of_birth_df: DataFrame, date_of_birth_col_name: str = "date_of_birth"
```

Comment:
Add docstring. Why does the date_of_birth_df contai? I assume it is a mapping in which case I would use date_of_birth_mapping.